misc/scripts/test-utils

```diff
@@ -8,6 +8,8 @@ declare -A util_vars=(
   [is_cheated]=0
   [is_installed]=0
   [non_interactive]=0
+  [num_threads]=0
+  [use_threads]=0
```

Comment:
LGTM, but I am guessing that these could be merged from a stylistic point of view? `num_threads = 1` implies `use_threads = 0`, so no need to have both. In this way we only provide the `-j` flag to `make` if `num_threads != 1`.